suite-native/transactions/src/components/TransactionDetail/TransactionDetailWarning.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Card, Text } from '@suite-native/atoms';
+import { Icon } from '@suite-common/icons';
+import { Translation } from '@suite-native/intl';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const cardStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundAlertRedSubtleOnElevation1,
+    gap: utils.spacings.small,
+    alignItems: 'center',
+
+    shadowColor: 'rgba(0, 0, 0, 0)',
```

Comment:
```suggestion
    ...utils.boxShadows.small,
```
The Android and iOS work differently with shadows. This utils styles ensures that the shadows are applied for both platforms. More here, if you are interested: https://blog.logrocket.com/applying-box-shadows-in-react-native/.

Reply:
But I want no shadow and there is a default shadow present. By setting the color to `rgba(0, 0, 0, 0)` I effectively make the shadow disappear

Reply:
Ah, I see. My bad.